jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -440,7 +440,7 @@ public long length()
                 set.add(s);
         }
         String[] result=set.toArray(new String[set.size()]);
-        Arrays.sort(result);
+        Arrays.sort(result); // TODO still necessary?
```

Comment:
https://github.com/eclipse/jetty.project/commit/1465e8ae19a878a571bafd5d6d62e3d3f7e7ce4f#diff-ca8a7c586007366cbd605eeecbd70d1bL408 no explanation beyond one-line commit comment.

Reply:
Probably not necessary... may remove but only in jetty-10

Reply:
@jglick can you take out the TODO comment?

Reply:
As you prefer.